hdbconnect_impl/src/conn/params/into_connect_params_builder.rs

```diff
@@ -62,6 +62,7 @@ impl IntoConnectParamsBuilder for Url {
             HDBSQL => false,
             HDBSQLS => true,
             _ => {
+                error!("unknown scheme: {}, from {}", self.scheme(), self);
```

Comment:
Traces will not be found in many cases, enriching the error message instead would be preferable.

`HdbError::UsageDetailed` allows using a String rather than just a &'static str:

```rust
                return Err(HdbError::UsageDetailed(format!(
                    "Unknown protocol '{}', only 'hdbsql' and 'hdbsqls' are supported",
                    self.scheme()
                )));
```

We cannot print out the complete URL, neither in the error message nor in the log::trace call, because the URL contains the db-user's password.